fmriprep/viz/config.json

```diff
@@ -2,14 +2,20 @@
 "sub_reports":
 [
     {
-        "name": "Anatomical",
+        "name": "Summary",
         "elements":
         [
             {
                 "name": "anat/summary",
                 "file_pattern": "anat/.*_summary",
```

Comment:
I think we can live with this :).

Additionally, we want #639 to be closed as soon as possible, and modifying `DerivativesDataSink` (which is currently working) may break it.